Plugins/Util/Util.cpp

```diff
@@ -126,6 +128,24 @@ NWNX_EXPORT ArgumentStack Hash(ArgumentStack&& args)
 }
 }
 
+NWNX_EXPORT ArgumentStack GetModuleMtime(ArgumentStack&&)
+{
+    CNWSModule *pMod = Utils::GetModule();
+    if (pMod->m_bIsSaveGame) {
+        LOG_DEBUG("NWNX_GetModuleMtime() module is a save game, returning 0");
+        return 0;
+    }
+    std::string modName = pMod->m_sModuleResourceName.SubString(12); // discard "CURRENTGAME:"
+    std::string modPath = Globals::ExoBase()->m_pcExoAliasList->ResolveFileName("MODULES:" + modName, Constants::ResRefType::MOD);
+    struct stat stat;
+    if (lstat(modPath.c_str(), &stat) == 0) {
```

Comment:
I think we'd want `stat` instead of `lstat`

From the [stat manpage](https://man7.org/linux/man-pages/man2/lstat.2.html):

```
       stat() and fstatat() retrieve information about the file pointed
       to by pathname; the differences for fstatat() are described
       below.

       lstat() is identical to stat(), except that if pathname is a
       symbolic link, then it returns information about the link itself,
       not the file that the link refers to.
```

Also bracket again!